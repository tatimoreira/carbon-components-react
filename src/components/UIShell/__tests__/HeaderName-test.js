import React from 'react';
import { mount } from 'enzyme';
import { HeaderName } from '../';

describe('HeaderName', () => {
  let mockProps;

  beforeEach(() => {
    mockProps = {
      className: 'custom-class',
      children: 'Name',
      prefix: 'IBM',
      href: '/',
    };
  });

  it('should render', () => {
    const wrapper = mount(<HeaderName {...mockProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
