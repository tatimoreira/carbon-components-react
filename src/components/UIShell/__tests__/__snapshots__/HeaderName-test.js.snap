// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderName should render 1`] = `
<HeaderName
  className="custom-class"
  href="/"
  prefix="IBM"
>
  <a
    className="bx--platform-header__name custom-class"
    href="/"
  >
    IBM
    Â 
    <span
      className="bx--platform-header__platform-name"
    >
      Name
    </span>
  </a>
</HeaderName>
`;
